api/__tests__/gh-utils.test.js

```diff
@@ -20,57 +20,58 @@
 
 'use strict';
 
-import { normalizeIssue, normalizeIssues } from '../src/libs/gh-utils';
-import { goodIssue, badIssue1, badIssue2, goodObject } from '../__fixtures__/utils-fixture';
+import { jsonReader } from '../src/libs/gh-utils/gh-helpers';
+import { ghHelper } from '../src/libs/gh-utils/gh-requests';
+import { goodIssue, goodObject } from '../__fixtures__/utils-fixture';
+import { mockedGHFn } from '../__mocks__/gh-fn';
 
-describe('normalizeIssue test', () => {
-  test('returns valid payload', () => {
-    expect(normalizeIssue(goodIssue)).toEqual({
-      issueId: 1,
-      realm: goodObject,
+describe('jsonReader test', () => {
+  test('read single key value', () => {
+    expect(jsonReader(goodObject, 'po')).toEqual(goodObject.po);
+  });
+
+  test('read array of key value', () => {
+    expect(jsonReader(goodObject, { id: 'id', owner: 'po' })).toEqual({
+      id: goodObject.id,
+      owner: goodObject.po,
     });
   });
 
-  test('throws when missing content', () => {
-    expect(() => normalizeIssue(badIssue1)).toThrow('Invalid GitHub issue.');
+  test('return original object when no keys', () => {
+    expect(jsonReader(goodObject)).toEqual(goodObject);
```

Comment:
FYI you can do snapshot testing with return values. That way you don't need to maintain `goodObject`. Its just like web:
```
const result = await someFuncitonCall();
expect(result).toEqualSnapshot();
```

Reply:
nice! I'll use snapshot when the output is not coming from the input.